src/main/java/org/minions/devfund/chelo/Squarelotron.java

```diff
@@ -0,0 +1,185 @@
+package org.minions.devfund.chelo.squarelotron;
+
+/**
+ * Class Squarelotron.
+ */
+public class Squarelotron {
+
+    public int[][] squarelotron;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename field "squarelotron" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make squarelotron a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)